gensim/utils.py

```diff
@@ -1005,15 +1004,13 @@ def pyro_daemon(name, obj, random_suffix=False, ip=None, port=None, ns_conf={}):
 
 def has_pattern():
     """
-    Function to check if there is installed pattern library
+    Function which returns a flag indicating whether pattern is installed or not
     """
-    pattern = False
     try:
-        from pattern.en import parse
-        pattern = True
+        import pattern
```

Comment:
Sure, will change it. Are there any changes required in the exception handling part?